capitoli/le_variabili.rst

```diff
@@ -0,0 +1,135 @@
+=======================
+Le Variabili
+=======================
+Il modo più semplice per farsi consegnare una scatola dal calcolatore, darle un nome e riempirla con un numero e’ quello di scrivere una frase come la seguente, che è chiamata .."todo: mettere testo in rosso"**“istruzione di assegnazione”:**            
+**SCATOLA1=7**
+Se la scatola di nome SCATOLA1 non è mai stata usata, Python crea una scatola nuova, la chiama SCATOLA1 e pone il numero 7 in quella scatola. Se invece la scatola di nome SCATOLA1 è già stata usata, Python toglie il suo vecchio contenuto e mette il numero 7 come suo nuovo contenuto.
+
+La scatola rappresenta quella che i matematici chiamano **variabile**.
+Questo nome ci ricorda che il contenuto di una scatola può cambiare da un momento all’altro.
+Ad essa e’ possibile assegnare un valore con l’istruzione di assegnazione.
+Il contenuto di una scatola può essere un numero, oppure un carattere o un messaggio (ossia una sequenza di caratteri).
+L’istruzione di assegnazione contiene un ordine per il calcolatore e costituisce un nuovo esempio di “istruzione”, ovvero un’operazione che Python capisce ed e’ in grado di eseguire.
+L’istruzione di assegnazione e’ costituita dal nome della scatola seguito dal segno =. 
+
+.. activecode:: esempio assegnazioni
+   :coach:
+   :caption: Come assegnare una variabile
+   
+
+   SCATOLA1 = 7
```

Comment:
Qui non è il caso di usare activecode in quanto non c'è niente in output. Usa un altro blocco